frontend/routes/viewerGui/components/views/components/viewItem/viewItem.component.tsx

```diff
@@ -138,6 +139,15 @@ export class ViewItem extends React.PureComponent<IProps, any> {
 		<Small>(Default View)</Small>
 	));
 
+	public renderViewpointNameWithShare = renderWhenTrue(() => (
+		<NameRow>
+			{this.renderViewpointName(true)}
+				<IconsGroup disabled={this.state.isDeletePending}>
+					{this.props.onShare && <StyledShareIcon onClick={this.handleShareLink} />}
```

Comment:
Can the dialog box either be made wider to fit the 260px name + share button, or make `Name` slightly shorter in the dialog box to make space for the share button.

At the moment, the share button gets pushed right by long view names and there's no spacing.

![Screenshot 2020-10-30 at 13 58 05](https://user-images.githubusercontent.com/1438332/97714135-afc28e80-1ab8-11eb-8635-0b0bc3a5833a.png)

Reply:
Thanks for spotting this one @Charence , I've already updated styling for views names in this specific case. It should look much better right now.

<img width="412" alt="Zrzut ekranu 2020-11-2 o 11 16 29" src="https://user-images.githubusercontent.com/24941866/97857101-86467480-1cfd-11eb-9411-e2baf6523eda.png">